Chat_App/Chat_App/UI/TypingComponentView.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  TypinComponentView.swift
+//  Chat_App
+//
+//  Created by Gi Oo on 26.04.23.
+//
+
+import UIKit
+
+class TypingComponentView: UIView {
+
+  // MARK: - Properties
+
+  private lazy var containerView = UIView()
+  private lazy var textView = UITextView()
+  private lazy var button = UIButton()
+  private let placeHolder: String
+  private let sendButtonImageName: String
+
+  // MARK: - Initializers
+
+  init(placeHolder: String, sendButtonImageName: String) {
+    self.placeHolder = placeHolder
+    self.sendButtonImageName = sendButtonImageName
+    super.init(frame: .zero)
```

Comment:
I noticed that super.init should be moved to line 23 to ensure that the superclass is initialized before accessing its properties or methods

Reply:
ვცადე გადავიტანე 23 ხაზზე და ამოაგდო ერორი :Property 'self.placeHolder' not initialized at super.init call